### DISCUSSION TITLE
border-color linear-gradient

### DISCUSSION THREAD

**Discussion by DartMitai** [davatorium/rofi#1820] [Q&A]
Can you tell the frame to set a gradient? I want to make the same gradient as the Hyprland default frame
`col.active_border = rgba(33ccffee) rgba(00ff99ee) 45deg
`

**Comment by DaveDavenport** [comment]
We currently do not support gradients on the border, only on the background of a widget.

You might be able to emulate it this  by adding no border but add a backgroun gradient to top level widget, pack another widget in the top-level widget with the real background color with some margin around it.

**Reply by DaveDavenport** [reply]
```bash
rofi -no-config -show drun -theme-str 'window { background-image: linear-gradient(45deg,pink,lightgreen); padding: 2em; } mainbox { background-color: blue; }'
```

**Reply by julianinsua** [reply]
This worked like a charm. Thanks!
